### ISSUE TITLE
Multi-labels?

### ISSUE THREAD

**Issue by Nedae** [NVlabs/stylegan2-ada-pytorch#165]
Is there any way to define multilabels for the images in the dataset? I already saw other issues such as #53 and #59 but no body provided the expected answer to those!
An example is when an image has more than one set of labels, such as "old/young", "happy/sad/neutral", and "man/woman". For instance, something like  below for `young happy man`?

```{
    "labels": 
        [
            ["00000/img00000000.png", 1, 0, 0]
        ]
}
```
Any part of the code to modify to support such a behaviour?

**Comment by Nedae** [comment]
From https://github.com/NVlabs/stylegan2-ada-pytorch/blob/d4b2afe9c27e3c305b721bc886d2cb5229458eba/dataset_tool.py#L62 it seems only single labels are supported!

**Comment by thusinh1969** [comment]
Gents,

Even for the multi-class problem, have you been able to produce non-streaks results (forget faces, we have enough of so cropped-focused face dataset !!!) ? I have NEVER been able to produce a good result at all, regardless of the size of the dataset (few dozen thousand to hundred thousand images) or kimg (30-50,000 kimg). Too many weird artifacts on generated images.

Any hint? Would R1 be a real reason that we have to seek to find that value? I used 1, 2, 5, 10. Same issue.

Steve

**Comment by GHBTM** [comment]
@[thusinh1969](https://github.com/thusinh1969) I have had much improve results with using gamma values (R1 regularization) in accordance with Appendix D (page 33) of the ADA paper... for a 32x32 grid had to drop gamma to 0.05 to 0.005, which scales almost linearly with resolution.  Check the paper out.  Tweaking learning rates should also help, but I don't (yet) have experience there.

**Comment by dongyunkim-arch** [comment]
I am also curious about multilabel...

**Comment by dongyunkim-arch** [comment]
@Nedae Hi Nedae, did you manage to work multi-labels styleGAN? I am working on the stuff, but stuck now..

**Comment by 49xxy** [comment]
> I am also curious about multilabel...

Have you got the solution？

**Comment by nom57** [comment]
I have got excellent results for my project on a small dataset of 278 images with consistent art style

the project would however benefit a lot from multi labels , would be cool if someone could make support for this

**Comment by 49xxy** [comment]
> 我的项目在一个包含 278 张图像的小数据集上获得了出色的结果，并且具有一致的艺术风格
> 
> 然而，该项目将从多标签中受益匪浅，如果有人可以对此提供支持会很酷

Hi！How do you set the multi-label of your dataset？
